src/main/java/com/code_intelligence/jazzer/agent/Agent.kt

```diff
@@ -138,9 +138,12 @@ fun installInternal(
     // "retransformed": They haven't been transformed yet.
     val classesToRetransform = instrumentation.allLoadedClasses
         .filter {
-            classNameGlobber.includes(it.name) ||
-                customHookClassNameGlobber.includes(it.name) ||
-                customHooks.additionalHookClassNameGlobber.includes(it.name)
+            !it.name.startsWith("com.code_intelligence.jazzer.") &&
```

Comment:
Could you add a comment explaining why this is needed? Just reference the failure we saw in CI, we don't have to root cause this to a concrete JDK bug.

Reply:
Added a comment.